### ISSUE TITLE
console warning App already provides property with key "ol-options"

### ISSUE THREAD

**Issue by marcomilone** [MelihAltintas/vue3-openlayers#341]
**Describe the bug**
```sh
22:02:41.911 runtime-core.esm-bundler.js:47 
[Vue warn]: App already provides property with key "ol-options". It will be overwritten with the new value.
warn$1 @ runtime-core.esm-bundler.js:47
provide @ runtime-core.esm-bundler.js:3974
install$3 @ vue3-openlayers.es.js:19471
use @ runtime-core.esm-bundler.js:3868
install @ vue3-openlayers.es.js:19695
use @ runtime-core.esm-bundler.js:3865
registerPlugins @ index.js:21
(anonymous) @ main.js:89
```

**Affected version(s)**

I've just upgraded 6.5.0 -> 8.0.0

**Package version(s)**

```
myapp@0.0.1 /workspaces/myapp
├── ol-contextmenu@5.4.0
├── ol-ext@4.0.18
├── ol@9.1.0
├── vue@3.4.25
└── vue3-openlayers@8.0.0
```

**To Reproduce**
Steps to reproduce the behavior, run:

vite

**Expected behavior**
no warning

**Additional context**
I guess is trying to set the ol-options twice, but I don't feel is my code doing it (I already check around, and before upgrade I had no warning).
Maybe this 2 lines?
- install$4 @ vue3-openlayers.es.js:18940
- install @ vue3-openlayers.es.js:19694

🖖🏻

**Comment by d-koppenhagen** [comment]
Already reported with #338. 
It's fixed but not released yet. I will publish a new version within the next days.

**Comment by marcomilone** [comment]
@d-koppenhagen 

Sorry for this superfluous issue

Delete it with no mercy

(before opening it ,I searched, but only in opened issues and #338 is already closed... 😛 )

**Comment by d-koppenhagen** [comment]
fixed with 8.1.0
